ui/raidboss/data/06-ew/trial/barbariccia-ex.ts

```diff
@@ -32,26 +42,60 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.aoe(),
     },
     {
-      id: 'BarbaricciaEx Hair Spray',
+      // Savage Barbery has 3 casts that all start at the same time.
+      // 5.7 duration: 7464, 7465, 7466, 7489, 748B, 7573 (all actual cast bar, unknown how to differentiate)
+      // 6.7 duration: 7574 (donut), 757A (line)
+      // 8.8 duration: 7575 (out, paired with donut), 757B (out, paired with line)
+      id: 'BarbaricciaEx Savage Barbery Donut',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '75A6', source: 'Barbariccia' }),
-      condition: Conditions.targetIsYou(),
-      infoText: (_data, _matches, output) => output.text!(),
+      netRegex: NetRegexes.startsUsing({ id: '7574', source: 'Barbariccia', capture: false }),
+      response: Responses.getIn(),
+    },
+    {
+      id: 'BarbaricciaEx Savage Barbery Line',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '757A', source: 'Barbariccia', capture: false }),
+      // TODO: should this say "side + out" or something else? out is possibly misleading.
+      response: Responses.getOut(),
```

Comment:
Maybe something like `out--away from indicator` or something to that effect? It's not exactly a tether, and it's not exactly arrows.

Reply:
Done.